crawler/config/centre.py

```diff
@@ -29,6 +29,13 @@
 CENTRE_REGEX_BARCODE = r"^[\W_]*([\w-]*)_([A-Z]\d{0,1}\d)[\W_]*$"
 CENTRE_DIR_BACKUPS = "data/backups"
 CENTRE_REGEX_SFTP_FILE_HERON = r"sanger_report_(\d{6}_\d{4}).*\.csv$"
+AP_SURVEILLANCE_1 = r"^AP-rna-\d+\.csv$"
+AP_SURVEILLANCE_2 = r"^[a-zA-Z]{2}-[a-zA-Z]{3}-\d+\.csv$"
+MK_SURVEILLANCE_1 = r"^RNA\d+\.csv$"
+MK_SURVEILLANCE_2 = r"^cpRNA\d+\.csv$"
```

Comment:
Is the only difference here the extra `cp`? This should be included in one regex instead with an or (`|`).